external/CMakeLists.txt

```diff
@@ -158,3 +157,6 @@ endif()
 # switch to a more primitive method, addr2line which does work.
 set(CPPTRACE_GET_SYMBOLS_WITH_ADDR2LINE ON CACHE BOOL "")
 add_subdirectory(cpptrace EXCLUDE_FROM_ALL)
+
+option(TRACY_ENABLE "Enable the profiler markers to be captured and displayable with the Tracy Profiler" OFF)
+add_subdirectory(tracy)
```

Comment:
```suggestion
if(TRACY_ENABLE)
    add_subdirectory(tracy)
endif()
```

Reply:
Is done in this commit: https://github.com/oxen-io/oxen-core/pull/1789/commits/360ce9b5d12e76457f1c4c9c8034b675221c4197